scripts/apps/archive/related-item-widget/relatedItem.js

```diff
@@ -70,6 +73,25 @@ function RelatedItemController(
         sort: [{versioncreated: 'desc'}]
     };
 
+    $scope.loading = true;
+    familyService.fetchRelatedItems($scope.item.event_id)
```

Comment:
imo would be better to have `item` as param to `fetchRelatedItems` and hide implementation that it's using `event_id` field